packages/utils/logger/lib/index.js

```diff
@@ -3,8 +3,8 @@
 const winston = require('winston');
 
 const formats = require('./formats');
-const createDefaultConfiguration = require('./default-configuration');
-const createOutputFileConfiguration = require('./output-file-configuration');
+const createDefaultConfiguration = require('./config/default-configuration');
+const createOutputFileConfiguration = require('./config/output-file-configuration');
```

Comment:
Would you mind adjusting the exports/requires to follow what has already been done for the formats pls?

`./configs/index.js`
```
module.exports = {
   default: require('./default-configuration'),
   outputFile: require('./output-file-configuration'),
};
```
And here
`const configs = require('./configs');`
`exports { ..., configs }`